src/gitchangelog/gitchangelog.py

```diff
@@ -1418,10 +1460,24 @@ def render_version(version):
         return s
 
     def render_commit(commit, opts=opts):
-        subject = commit["subject"]
+        # Get Jira summary
+        ticket = RE_TICKET.search(commit["subject"])
+        ticket = ticket.group()[1:-1] if ticket else None
+        if ticket:
+            try:
+                issue = jira.issue(ticket, fields="summary")
+                subject = "[[{}]]({}) {}".format(
```

Comment:
there is one pair of extra brackets 
Output:
`- [[KLTB002-14238]](https://kolibree.atlassian.net/browse/KLTB002-14238) Setup repository for changelog`

Expected:
`- [KLTB002-14238](https://kolibree.atlassian.net/browse/KLTB002-14238) Setup repository for changelog`

```suggestion
                subject = "[{}]({}) {}".format(
```

Reply:
Ok, thought that we want Jira link visually inside square brackets :) Will remove them.

Reply:
Updated. Lets please keep notes of any other formatting glitch/issue/... and tackle it in following formatting task.